Makefile

```diff
@@ -6,4 +6,5 @@ image:
 	docker build --rm -t $(IMAGE_NAME) .
 
 test: image
-	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && bundle exec rake"
+	docker run --rm $(IMAGE_NAME) sh -c "cd /usr/src/app && \
+		CODECLIMATE_REPO_TOKEN=ca5120f90202a99c43c0914d301a94e5908136d146ecc0d7fe7fde1c3f9d325e bundle exec rake"
```

Comment:
Since you're touching this anyway, you can make it

```console
docker run --rm \
  --env CODECLIMATE_REPO_TOKEN
  --workdir /usr/src/app
  $(IMAGE_NAME) bundle exec rake
```